helpers/helper-translation-discussion.php

```diff
@@ -510,6 +510,27 @@ public function sanitize_translation_id( $translation_id ) {
 		}
 		return $translation_id;
 	}
+
+	/**
+	 * Kills WordPress execution and displays HTML page with an error message if the original id is incorrect.
+	 *
+	 * Used as callback to validate the original_id passed on rejecting a string with feedback
+	 *
+	 * @since 0.0.2
+	 *
+	 * @param int|string $original_id   The id of the original for the rejected translation.
+	 *
+	 * @return int|string
+	 */
+	public function sanitize_original_id( $original_id ) {
+		if ( ! is_numeric( $original_id ) ) {
+			if ( $original_id > 0 && ! GP::$original->get( $original_id ) ) {
+				wp_die( 'Invalid original ID' );
+			}
+		}
```

Comment:
I think it makes sense to check first if the $orginal_id is numeric and then if it exists in the database.
```suggestion
		if ( ! is_numeric( $original_id ) ) {
			wp_die( 'Invalid ID format' );
		} elseif ( $original_id > 0 && ! GP::$original->get( $original_id ) ) {
				wp_die( 'Invalid original ID' );
		}
```

Reply:
Thanks, I would remove the is_numeric check and just do `if ( $original_id > 0 && ! GP::$original->get( $original_id ) )` this was addressed in another PR